NEWS.rst

```diff
@@ -6,3 +6,8 @@ Internal changes
 
 * #23992: Updated API documentation and README
 * #23993: Reorganisation of source code layout
+
+Backend changes
+---------------
+
+* #22849: Implemented support for signed SAML AuthN Requests.
```

Comment:
The release notes are for humans, so we should write out “authentication” in full :)

Reply:
AuthN Request is a specific piece of SAML terminology, so I reused the name here to make it clear what I'm referring to.